README.md

```diff
@@ -20,6 +20,8 @@ Studio -> Manage Packages -> Community -> (Search) UiPathTeam.Extensions.Activit
 
 * `Dictionary Contains Value`: Checks if the given value is present in an existing Dictionary<Object, Object> and returns Boolean value to confirm if present or not.
 
+* `Dictionary To String`: Converts a dictionary to a string. Note that this only converts simple objects. Additional deconstruction is required for complex data types (if your dictionary has dictionaries for elements, they will not be printed).
+
```

Comment:
what does additional deconstruction mean to an end user?  how will they accomplish this?